docs/fastdds/dynamic_types/dynamic_types.rst

```diff
@@ -1,13 +1,13 @@
 .. _dynamic-types:
 
-Dynamic Topic Types
-===================
+XTypes 1.3
+==========
 
-.. _DDS-XTypes V1.2: http://www.omg.org/spec/DDS-XTypes/1.2
+.. _DDS-XTypes V1.3: http://www.omg.org/spec/DDS-XTypes/1.3
 
 eProsima Fast DDS provides a dynamic way to define and use topic types and topic data.
 Our implementation follows the *OMG Extensible and Dynamic Topic Types for DDS interface*.
-For more information, you can read the specification for `DDS-XTypes V1.2`_.
+For more information, For more information, refer to the specification for `DDS-XTypes V1.3`_.
 
 The dynamic topic types offer the possibility to work over RTPS without the restrictions related to the IDLs.
```

Comment:
This is a resume of Dynamic Language Binding. I will call it in this way and not "Dynamic Topic Types". Also I will include a resume about the discovering matching.

Reply:
fixed in [e6714a3](https://github.com/eProsima/Fast-DDS-docs/pull/730/commits/e6714a3c5febe9d35a953b5c4f57fb084ed94a81)